raiden/tests/fixtures/abi.py

```diff
@@ -6,6 +6,7 @@
     CHANNEL_MANAGER_ABI,
     NETTING_CHANNEL_ABI,
     REGISTRY_ABI,
+    DECODER_TESTER_ABI,
```

Comment:
I suggest we keep the test abi out of the main source tree -- see also below, the ABI/the ABI fixture is nowhere used